app/src/main/res/layout/message.xml

```diff
@@ -50,12 +48,13 @@
 
                     <TextView
                         android:id="@+id/message_user"
-                        android:layout_width="wrap_content"
+                        android:layout_width="0dp"
```

Comment:
Does the name of the user actually correctly appear in the message view now with this change?

Reply:
When android studio proposed this change, I also asked myself this question because it seems a bit strange xD. But there is visibly no problem and this would serve according to android studio just for optimization concerns